InfinispanAPIObject.adoc

```diff
@@ -0,0 +1,36 @@
+== Infinispan API
+
+=== Motivation
+
+Being able to offer a simple API to create clustered caches and improve API usability
+
+
+== Infinispan
+
+This factory will help to create a cluster of Infinispan.
+
+Whenever a new member is added on the VM, the implicit configuration will handle the visibility between members.
+
+```java
+
+public final class Infinispan {
+
+    private Infinispan() {}
+
+    //Constructs and starts a new instance of the CacheManager, using the system defaults.
+    public static EmbeddedCacheManager createClusteredCache() {}
```

Comment:
@tristantarrant Why would you want to have a duplicate instances? I see `Infinispan` as just a wrapper over CM, the instance should not have any state on its own.